app/mailers/user_mailer.rb

```diff
@@ -353,11 +353,11 @@ def prompter_notification(work_id, collection_id=nil)
   def delete_work_notification(user, work)
     @user = user
     @work = work
-    work_copy = generate_attachment_content_from_work(work)
-    work_copy = ::Mail::Encodings::Base64.encode(work_copy)
-    filename = work.title.gsub(/[*:?<>|\/\\\"]/,'')
-    attachments["#{filename}.txt"] = { content: work_copy, encoding: "base64" }
-    attachments["#{filename}.html"] = { content: work_copy, encoding: "base64" }
+    @download = Download.new(@work, mime_type: "text/html")
+    @download.generate
```

Comment:
Download writes to a file, and then we have to `File.read` from it. I think the part that we need is:
https://github.com/otwcode/otwarchive/blob/5161fd10c15abd943d9c50158c06aa0fe55e420d/app/models/download_writer.rb#L23-L34
That gives the string right away. You'll need to refactor a little to get a separate public method for it, then you can do something like:
```
DownloadWriter.new(@download).html
```